connexta/eve
# EVE49: Update Github Component

#### What does this PR do?
Updated PR component to show more information about each PR and cycle through them automatically.  You can now click on a PR to be taken to its GitHub page or click the numbers on the sidebar to see a different PR.

#### Who is reviewing it?

@vinamartin @Kjames5269 @bankent1 @alchucam @mcalcote @haydenudelson

#### How should this be tested?
`yarn build` and `yarn start`. Click on different PRs to navigate to GitHub, click on the sidebar to navigate to different PR's.  If you'd like, you can change the path that gets passed into the component to see different repositories.

#### Screenshots
<img width="1234" alt="newgithubcomponent" src="https://user-images.githubusercontent.com/20527865/62084829-0b93cd00-b20f-11e9-8395-b4de6656c32c.PNG">



#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
